bower.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "stencil-progress-bar",
+  "description": "",
+  "authors": [
+    "Mobify"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/mobify/stencil-progress-bar.git"
+  },
+  "keywords": [
+    "mobify",
+    "components",
+    "patterns"
+  ],
+  "license": "MIT",
+  "main": [
+    "progress-bar.dust"
+  ],
+  "ignore": [
+    ".editorconfig",
+    ".gitignore",
+    "package.json",
+    "Gruntfile.js",
+    "Gemfile",
+    "Gemfile.lock",
+    "tests"
+  ],
+  "dependencies": {
+    "stencil-variables": "mobify/stencil-variables#*",
+    "stencil-utils": "mobify/stencil-utils#dev",
+    "level": ">=0.1.0",
+    "spline": "~1.0.1",
+    "font-awesome": "~4.3.0",
+    "zeptojs": "1.1.6",
```

Comment:
This raises a question that we need to address for Stencil as a whole: what selector library they should depend on. Adaptive 2.0+ supports using both Zepto and jQuery, with jQuery as the default. I think we should use jQuery for the component’s own tests, and it should work with both jQuery. Need to double-check, but I believe with Zepto Adaptive uses a custom build of Zepto with at least the data module built in. Certainly I think we should avoid components depending on custom builds of Zepto.